docs/content/preview/yugabyte-platform/back-up-restore-universes/restore-ysql-single-table.md

```diff
@@ -0,0 +1,132 @@
+---
+title: Restore a single YSQL table
+headerTitle: Restore a single YSQL table
+linkTitle: Restore a single YSQL table
+description: Use YugabyteDB Anywhere to restore data.
+headContent: Restore from full or incremental backups
+menu:
+  preview_yugabyte-platform:
+    parent: restore-universe-data
+    identifier: restore-ysql-single-table
+    weight: 30
+type: docs
+---
+
+Currently, YugabyteDB Anywhere supports backing up and restoring entire YSQL databases only. However, by performing some manual steps, you can restore single tables to a YSQL database using the following tools:
+
+- [ysql_dump](../../../manage/backup-restore/export-import-data/) - exports data from a YSQL database into a SQL script file.
+- [ysqlsh](../../../admin/ysqlsh/) - YugabyteDB SQL shell for interacting with YugabyteDB using YSQL.
+
+Suppose you have a problem with a single table in a database called source_db, and you want to restore the table from a backup taken previously. You would perform the following steps:
+
+1. Using a backup of source_db, restore the backup to a new database (call it restored_db). This can be done on the same or a different universe.
+1. In restored_db, use ysql_dump to export an SQL script file of the table to be restored.
+1. In source_db, use ysqlsh to drop the table with the same name (that is, the table you want restored from the backup), if it exists. (You only need to do this if the table with the same name exists on the database.)
```

Comment:
What do you mean by if the table exists? I thought we were talking about restoring this "problematic" table. So it should exist right?

Reply:
The problem might be the table was accidentally dropped.